test/html_generator_test.dart

```diff
@@ -60,7 +60,7 @@ void main() {
     test('header and footer', () {
       String content = templates.indexTemplate({},
           assumeNullNonExistingProperty: true, errorOnMissingProperty: false);
-      expect(content.contains('<p>User inserted message</p>'), true);
+      expect(content.contains('<p>User inserted message</p>'), false);
```

Comment:
for next time: isFalse

or:

expect(content, contains(...))

(the error messages are nicer this way)